docs/metrics.md

```diff
@@ -17,6 +17,7 @@ Measurement accuracy can be impacted by other parallel processing that might be
 ## Change log
 This list is to help notify if there are additions, changes or removals to metrics. Latest changes are at the top of this list.
 
+- Remove ovnkube_controller_ovn_cli_latency_seconds metrics since we have moved most of the OVN DB operations to libovsdb.
```

Comment:
nit: It needs to be moved down to the end of this list and need to link to PR number. 

It can be a FUP since anyways we have it here for history so we can dig out rest of the details from git commit if needed.